quickfixj-core/src/main/java/quickfix/Session.java

```diff
@@ -2333,8 +2333,13 @@ private void nextLogon(Message logon) throws FieldNotFound, RejectLogon, Incorre
     }
 
     private void resendMessages(Message receivedMessage, int beginSeqNo, int endSeqNo)
-            throws IOException, InvalidMessage, FieldNotFound {
-
+            throws IOException, FieldNotFound {
+        // Prevent a one-time load of data that is too large, resulting in memory OOM。 if config resendRequestChunkSize。
```

Comment:
Could you please check that you don't use strange characters in the source code, e.g. as after
```
resendRequestChunkSize。
```

Reply:
No, what I optimized was the problem of overloading at one time.